### PR TITLE
winston removed

### PR DESCRIPTION
**Pull request by jifeon**
I'm replacing winston with [pino](https://github.com/pinojs/pino) in our repositories because it's:

- faster, all log transformations are in separate processes
- has no memory leaks
- more natural in the way how it's working - just json logger
- fresh winston has strange behaviour against arguments you can pass and logging errors

Storing instantiating of logger right inside repo must more useful, that's why plyo.tools knows nothing about logger anymore

### CODE REVIEW

**Comment by jifeon** [comment]
[Possible memory leak in API](https://trello.com/c/2cxUIODW/1468-possible-memory-leak-in-api)

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "private": true,
-  "version": "3.1.5",
+  "version": "3.2.0",
```
Comment (frenzzy) [inline package.json]: Removing API must be a breaking change, isn't it?

Comment (jifeon) [reply package.json]: yes, you're right. 4.0

**Review by AGresvig — APPROVED** [review]

**Comment by jifeon** [comment]
[Possible memory leak in API](https://trello.com/c/2cxUIODW/1468-possible-memory-leak-in-api)

File: package.json (hunk 1 above)
Comment (jifeon) [reply package.json]: 4.0 is published

**Pull request merged**